src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import ball.characteristics.Ball;
+import java.util.Random;
+
+public class Lottery {
+    public static String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(ColorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
Instead of creating a new instance of `Random` every time the method is called, consider defining it as a private static final field in the class. This will be more efficient as you will reuse the same `Random` instance.

Reply:
not fixed